### PR TITLE
Replace (some) of First & Last Name variable example references (Batch 3)

### PR DESCRIPTION
**Pull request by hergaiety**
## Issue
https://github.com/ember-learn/guides-source/issues/1480
Prior batch: https://github.com/ember-learn/guides-source/pull/1537
_This is the last batch!_

TL;DR Replacing `firstName` and `lastName` example code with `title` and `name`, or just simply `name`, or something else entirely for i18n/W3C/respect concerns.

## Pages touched in this batch of changes
* [x] [In-Depth Topics - Native Classes In-Depth - Fields](https://guides.emberjs.com/release/in-depth-topics/native-classes-in-depth/#toc_fields)
* [x] [Upgrading - Glimmer Components - Arguments](https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_arguments)
* [x] [Upgrading - Native Classes](https://guides.emberjs.com/release/upgrading/current-edition/native-classes/)
* [x] [Models - Defining Models - Defining Attributes](https://guides.emberjs.com/release/models/defining-models/#toc_defining-attributes)

## Notes

I don't feel this needs to go back prior to `/release` but I'm open to a suggested amount of backporting.

Open to suggested alternatives for these changes as well.

### CODE REVIEW

**Review by locks — COMMENTED** [review]
Seems like solid work from a quick overview. I'll do a more detailed review soon if someone doesn't beat me to it.

File: guides/release/upgrading/current-edition/glimmer-components.md [hunk 1]
```diff
@@ -433,17 +433,17 @@ After:
 ```js
 import Component from '@glimmer/component';
 
-export default class PersonComponent extends Component {
-  get firstName() {
-    return this.args.firstName || 'Kenneth';
+export default class ImageComponent extends Component {
+  get width() {
+    return this.args.width === undefined ? this.args.width : 0;
```
Comment (ijlee2) [inline guides/release/upgrading/current-edition/glimmer-components.md:438]: Let's use the nullish coalescing operator? (We'll assume that the developer knows how to set their `configs` file to transpile the operator if necessary.)

```suggestion
    return this.args.width ?? 0;
```

File: guides/release/upgrading/current-edition/glimmer-components.md [hunk 2]
```diff
@@ -433,17 +433,17 @@ After:
...
   }
 
-  get lastName() {
-    return this.args.lastName || 'Larsen';
+  get height() {
+    return this.args.height === undefined ? this.args.height : 0;
```
Comment (ijlee2) [inline guides/release/upgrading/current-edition/glimmer-components.md:442]:
```suggestion
    return this.args.height ?? 0;
```

File: guides/release/upgrading/current-edition/native-classes.md [hunk 3]
```diff
@@ -263,20 +263,22 @@ function join(...keys) {
 
 // Before
 const ClassicPerson = EmberObject.extend({
-  firstName: 'Katie',
-  lastName: 'Gengler',
+  nickName: 'Tom',
```
Comment (ijlee2) [inline guides/release/upgrading/current-edition/native-classes.md:266]: A nitpick:
```suggestion
  nickname: 'Tom',
```

File: guides/release/upgrading/current-edition/native-classes.md [hunk 4]
```diff
@@ -263,20 +263,22 @@ function join(...keys) {
...
+  title: 'Prof.',
+  name: 'Tomster',
 
-  fullName: join('firstName', 'lastName'),
-  name: alias('fullName'),
+  fullName: join('title', 'name'),
+  displayName: alias('nickName'),
```
Comment (ijlee2) [inline guides/release/upgrading/current-edition/native-classes.md:271]:
```suggestion
  displayName: alias('nickname'),
```

File: guides/release/upgrading/current-edition/native-classes.md [hunk 5]
```diff
@@ -263,20 +263,22 @@ function join(...keys) {
...
-  firstName = 'Katie';
-  lastName = 'Gengler';
+  nickName = 'Tom';
+  title = 'Prof.';
+  name = 'Tomster';
 
-  @join('firstName', 'lastName') fullName;
-  @alias('fullName') name;
+  @join('title', 'name') fullName;
+  @alias('nickName') displayName;
```
Comment (ijlee2) [inline guides/release/upgrading/current-edition/native-classes.md:281]:
```suggestion
  @alias('nickname') displayName;
```

**Review by ijlee2 — APPROVED** [review]
@hergaiety Thank you for completing the issue! 🎉

While the changes can be merged as they are now, I think it'd help if we apply my suggestions beforehand. Will you have a look?

**Comment by hergaiety** [comment]
@ijlee2 suggestions applied, many thanks!

**Pull request merged**